android/library/src/main/java/io/logto/client/config/LogtoConfig.kt

```diff
@@ -9,7 +9,8 @@ data class LogtoConfig(
 ) {
     val encodedScopes: String = scopes.joinToString(" ")
 
-    val cacheKey: String = "$clientId::$encodedScopes"
+    val cacheKey: String =
+        java.net.URLEncoder.encode("$clientId::$encodedScopes", "utf-8").replace("+", "%20")
```

Comment:
we could import `URLEncoder` and use `URLEncoder.encode`.
`java.net` is verbose